__main__.py

```diff
@@ -66,33 +66,36 @@ def validate_project(platform, project, report_type):
 
 def args_to_list(platform, projects):
     projects_list = []
-    # we need to convert projects data, if str,  to a list  (if not already)
-    if isinstance(projects, str):
-        if projects == 'all':
-            if platform == 'desktop':
-                for project in PROJECTS_DESKTOP[:-1]:
-                    projects_list.append(project)
-
-            if platform == 'mobile':
-                for project in PROJECTS_MOBILE[:-1]:
-                    projects_list.append(project)
-
-            if platform == 'ecosystem':
-                for project in PROJECTS_ECOSYSTEM[:-1]:
-                    projects_list.append(project)
-        else:
-            projects_list = [projects]
+
+    if platform != '' and projects != '':
+        platform = platform.lower()
```

Comment:
I moved these into a conditional to prevent them from breaking when empty

Reply:
actually this is not needed...if it is empty, it can not be lower...